src/TouristInfoCards.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+const TouristInfoCards = () => {
```

Comment:
This component is responsible for rendering all of the tourist info cards.

Perhaps we have a component that is responsible for rendering just a single card? What props would you pass to this component?

Reply:
Nothing , Do you mean I should make one card and props transfer the data which is about three cards in the component?